### ISSUE TITLE
error during demo profile install

### ISSUE THREAD

**Issue by tpham00** [istio/istio#19102]
**Bug description**
I'm getting this error when running "istioctl manifest apply --set profile=demo":
Component IngressGateway failed install:
========================================
Error: error running kubectl: exit status 1

Error detail:

_**Error from server (NotFound): error when creating "STDIN": the server could not find the requested resource (post gatewaies.networking.istio.io) (repeated 1 times)**_

**Expected behavior**

**Steps to reproduce the bug**
Following https://istio.io/docs/setup/getting-started/
**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**
$ istioctl version --remote
client version: 1.4.0
control plane version: 1.4.0
data plane version: none
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.1", GitCommit:"b1b29978270dc22fecc592ac55d903350454310a", GitTreeState:"clean", BuildDate:"2018-07-18T11:37:06Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:14:56Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}

**How was Istio installed?**
https://istio.io/docs/setup/getting-started/
**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by howardjohn** [comment]
> Error from server (NotFound): error when creating "STDIN": the server could not find the requested resource (post gatewaies.networking.istio.io) (repeated 1 times)

did you make a typo here or was it actually logging `gatewaies.networking.istio.io` (rather than `gateways.networking.istio.io`)?

Anyhow seems like the CRD is not created properly

**Comment by rvennam** [comment]
We've seen this before...https://github.com/istio/istio/issues/8096

Can you update your kubectl version and try again?

**Comment by tpham00** [comment]
Specifically what version of kubectl?

**Comment by tpham00** [comment]
Im running:
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.1", GitCommit:"b1b29978270dc22fecc592ac55d903350454310a", GitTreeState:"clean", BuildDate:"2018-07-18T11:37:06Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}

**Comment by rvennam** [comment]
Your server is 1.14, so I recommend 1.14 or higher. https://kubernetes.io/docs/tasks/tools/install-kubectl/

**Comment by tzakrajs** [comment]
This is happening to me on 1.16.3 (Ubuntu 18.04 with fresh kubeadm init)

The manifests used below were generated by istioctl manifest generate from binaries shipped with istio 1.4.0:
```
$ kaf gateway-crd.yml
customresourcedefinition "gateways.networking.istio.io" configured
$ kaf ingressgateway.yml
Error from server (NotFound): error when creating "ingressgateway.yml": the server could not find the requested resource (post gatewaies.networking.istio.io)
```

**Comment by jasonwzm** [comment]
@tzakrajs are you still seeing the issue? I tried on my local and could not reproduce the issue.
